components/social-media-icon/index.js

```diff
@@ -0,0 +1,31 @@
+import { IconMapper } from './social-media.constant';
```

Comment:
follow import sequence

Reply:
as mentioned, "sort import" VS code extension takes care of industry standard import sequence

Reply:
still messing the imports, this is NIT. Not a major bug.

[official react code](https://github.com/facebook/react/blob/master/fixtures/flight/src/index.js)